src/newrelic/app_test.go

```diff
@@ -59,10 +60,14 @@ func TestConnectPayloadInternal(t *testing.T) {
 	}
 
 	expected.Util.Hostname = info.Hostname
+	utilization.OverrideDockerId(expected.Util, info.DockerId)
 
 	pid := 123
 	b := info.ConnectPayloadInternal(pid, util)
 
+	expectedDockerId, _ := utilization.GetDockerId(expected.Util)
```

Comment:
```suggestion
	expectedDockerId, _ := "1056761e1f44969c959364a8e26e9345b37ccb91aef09a8173c90cf1d1d99156"
```

Reply:
I think if we go this route we aren't really testing anything different from the `TestConnectPayloadInternalDocker` test

Reply:
See c28cb6775c1128750e7bb9b523fd7b1164bfdd52